latex/Research_Strategy.tex

```diff
@@ -127,16 +127,17 @@
 
 %Promise of physical methods, how they could transform drug discovery and the design of new small molecules for chemical biology
 % (If space permits here we will allude also to how this could help with broader areas too such as engineering of biologics)
-\textbf{Physical methods are poised to transform drug discovery and chemical biology by enabling true molecular design.}
-While modeling work is used extensively in drug discovery, its main role at present is to aid with idea generation or filter large libraries of compounds for screening. 
+\textbf{Physical modeling is poised to transform drug discovery and chemical biology by enabling true molecular design.}
+While modeling  is used extensively in drug discovery, its main role at present is to aid with idea generation or filter large libraries of compounds for screening. 
 Instead, we imagine using computational techniques extensively to guide the design process. 
-Consider a medicinal chemist in the not-too-distant future who has just finished synthesizing several new derivatives of an existing inhibitor as potential drug leads targeting a particular biomolecule, and has obtained binding affinity or potency data against the desired biomolecular target. 
-Before leaving work, he or she generates ideas for perhaps 100 new compounds which could be synthesized next, then sets a computer to work overnight. 
-By morning, the compounds have all been prioritized based on reliable predictions of their affinity for the desired target, selectivity against alternative targets which should be avoided, solubility, and membrane permeability.  
-The chemist then looks through the predicted properties for the top few compounds and selects some for synthesis. 
+Consider a medicinal chemist in the not-too-distant future who has just finished synthesizing several new derivatives of an existing inhibitor, and has obtained binding affinity or potency data against the desired biomolecular target. 
+Before leaving work, she generates ideas for perhaps 100 new compounds which could be synthesized next, setting her computer to work overnight. 
+By morning, the idea compounds have been prioritized based on reliable predictions of their affinity for the desired target, selectivity against antitargets, solubility, and membrane permeability.  
+{\color{red}[JDC: Should we say something about pharmacodynamics/kinetic and toxicity liabilities? We \emph{are} starting with HSA, after all.]}
```

Comment:
I like how your "trimming for length and impact" results in suggestions for what to add. ;)